Services/service_response.h

```diff
@@ -19,6 +19,7 @@
 
 #include "services.h"
 
+SAT_returnState service_response_task(void *param);
```

Comment:
Does this need to be publicly accessible? If not, i'd propose just leaving it out.